### ISSUE TITLE
Library version updates (***After Midwest***)

### ISSUE THREAD

**Issue by borkae** [FRC1732/2020-Competition#102]
- [ ] Spark Max firmware to 1.5.2 via SparkMax client tool
- [ ] Spark Max vendor library to 1.5.2 by reloaded venderlibs - Check for updates (online) - I show all 4 libraries needing updates
- [ ] Update the Gradle plugin - https://plugins.gradle.org/plugin/edu.wpi.first.GradleRIO

**Comment by SamuelStone-beep** [comment]
Rev fixed the issue about not having library support for the motor controller firmware, right?

**Comment by borkae** [comment]
Yes.  both firmware and library are at the same version (1.5.2) and locally I am not having problems.  As I recall the release notes on the 1.5.2 have a few neat features for troubleshooting some problems.
